site-src/v1alpha2/references/policy-attachment.md

```diff
@@ -0,0 +1,290 @@
+Gateway API. Instead, implementations may choose to represent these concepts
+with custom Policy resources.
+
+Policies attached to Gateway API resources and implementations must use the
+following approach to ensure consistency across implementations of the API.
+There are three primary components of this pattern:
+
+* A standardized means of attaching policy to resources.
+* Support for configuring both default and override values within policy
+  resources.
+* A hierarchy to illustrate how default and override values should interact.
+
+This kind of standardization not only enables consistent patterns, it allows
+future tooling such as kubectl plugins to be able to visualize all policies that
+have been applied to a given resource.
+
+## Policy Attachment for Ingress
+Attaching policy to Gateway resources for ingress use cases is relatively
+straightforward. A policy can reference the resource it wants to apply to.
+Access is granted with RBAC - anyone that has access to create a RetryPolicy in
+a given namespace can attach it to any resource within that namespace.
+
+![Simple Ingress Example](/images/policy/ingress-simple.png)
+
+To build on that example, it’s possible to attach policies to more resources.
+Each policy applies to the referenced resource and everything below it in terms
+of hierarchy. Although this example is likely more complex than many real world
+use cases, it helps demonstrate how policy attachment can work across
+namespaces.
+
+![Complex Ingress Example](/images/policy/ingress-complex.png)
+
+## Policy Attachment for Mesh
+Although there is a great deal of overlap between ingress and mesh use cases,
+mesh enables more complex policy attachment scenarios. For example, users may
+want to apply policy to requests from a specific namespace to a backend in
+another namespace.
+
+![Simple Mesh Example](/images/policy/mesh-simple.png)
+
+Policy attachment can be quite simple with mesh. Policy can be applied to any
+resource in any namespace but it can only apply to requests from the same
+namespace if the target is in a different namespace.
+
+At the other extreme, policy can be used to apply to requests from a specific
+workload to a backend in another namespace. A route can be used to intercept
+these requests and split them between different backends (foo-a and foo-b in
+this case).
+
+![Complex Mesh Example](/images/policy/mesh-complex.png)
+
+## Target Reference API
+
+Each Policy resource MUST include a single `targetRef` field. It MUST not
+target more than one resource at a time, but it can be used to target larger
+resources such as Gateways or Namespaces that may apply to multiple child
+resources.
+
+The `targetRef` field MUST be an exact replica of the `PolicyTargetReference`
+struct included in the Gateway API. Where possible, we recommend using that
+struct directly instead of duplicating the type.
+
+### Policy Boilerplate
+The following structure can be used as a starting point for any Policy resource
```

Comment:
```suggestion
The following structure must be used for any Policy resource
```

Reply:
We should then in the code comments document which field can be omitted from a specific resource.

Reply:
> in the code comments document which field can be omitted from a specific resource.

You have done this to a certain extent in the document itself below.

Reply:
Updated the comment + added some guidance that policy resources could choose to omit one of the "Override" or "Default" fields.